src/Microsoft.IdentityModel.Tokens/SecurityTokenDescriptor.cs

```diff
@@ -119,5 +119,13 @@ public class SecurityTokenDescriptor
         /// values will be overridden.
         /// </summary>
         public ClaimsIdentity Subject { get; set; }
+
+        /// <summary>
+        /// Gets or sets a bool that controls if token creation will set default 'cty' if not specified.
+        /// <remarks>
+        /// Applies to only JWT tokens.
+        /// </remarks>
+        /// </summary>
+        public bool SetDefaultCtyClaim { get; set; } = true;
```

Comment:
Since the 'Cty' header claim is related to Encryption, if we put the property on EncryptingCredentials, we may not need the additional public overloads that pass the bool: 'setDefaultCtyClaim'.

Reply:
Makes sense. Moved it to EncryptingCredentials.